frontend2/src/components/BaseCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+interface BaseCardProps {
+  GameImage: string;
```

Comment:
let's change all these properties to camelCase

`GameImage` => `gameImage`

etc